packages/fixie-sdk/src/conversation.tsx

```diff
@@ -0,0 +1,52 @@
+/// <reference lib="dom" />
+import * as AI from 'ai-jsx';
+import { GetConversationResponse } from './types.js';
+import { AssistantMessage, FunctionCall, FunctionResponse, UserMessage } from 'ai-jsx/core/conversation';
+import { fixieContext } from './request-wrapper.js';
+
+export async function FixieConversation(_: {}, { getContext }: AI.ComponentContext) {
+  const fixieContextValue = getContext(fixieContext);
+  if (!fixieContextValue) {
+    throw new Error('FixieConversation components may only be used in the context of requests from Fixie.');
```

Comment:
If we were in AI.JSX, it would be nice to use the AI.JSX more structured errors here. (Which I know you also don't see a ton of value in. 🤪 )